QuizMakerAngular/src/app/components/chat-inbox/chat-inbox.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit, Input } from '@angular/core';
+import * as io from 'socket.io-client'
+import { AuthService } from "../../services/auth.service";
+
+const SOCKET_ENDPOINT = 'localhost:3001';
```

Comment:
We need to look at where this is when hosted and probably move to the config files

Reply:
Yeah, that was the plan, but rather you deal with that!